package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stylelint",
-  "version": "14.10.0",
+  "version": "14.11.0",
```

Comment:
Is it possible to prevent updating `package.json`? I'm concerned about conflict with `np`.

Reply:
Let me check.